src/Pages/ProductPage/ProductPage.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import "./ProductPage.css"
+import { useParams } from "react-router-dom";
+
+function ProductPage()
+{
+    const { id } = useParams()
+    const productDetailsOnStorage = localStorage.getItem(`${id}`)
+    const productDetails = JSON.parse(productDetailsOnStorage)
+    const {
+        bookname,
+        author,
+        originalprice,
+        discountedprice,
+        discountpercent,
+        imgsrc, 
+        imgalt,
+        badgetext, 
+        outOfStock,
+        rating, 
+        description
```

Comment:
either use camelCase or capital alphabets text styling for all the props for readibility.

Reply:
Done.